src/expression.h

```diff
@@ -28,7 +28,7 @@ class UnaryOp : public Expression
 		Not,
 		Negate
 	};
-
+    virtual bool isLiteral() const { return true;}
```

Comment:
@amarjeetkapoor1 This looks wrong, unless I've misunderstood something.
This will currently allow 'B' to be a customizer variable:
```
A = 10;
B = !(A == 5);
```